installation/linux.md

```diff
@@ -97,10 +97,11 @@ Decide between two options:
   sudo apt-get install openhab2
   ```
 
-  The lastest snapshots will download add-ons on request, if you plan on disconnecting your machine from the internet then you will want to also install the addons package:
+  The latest snapshots will download add-ons on request, if you plan on disconnecting your machine from the internet then you will want to also install the add-ons package. Optionally, you can also install the legacy add-ons package, which contains 1.x bindings for compatibility (useful if you're [coming from openHAB 1.x]({{base}}/tutorials/migration.html) for example):
```

Comment:
You could also point to http://docs.openhab.org/addons/bindings.html#legacy-1x-bindings here.

Besides that I am not going to talk about "one sentence per line" again :panda_face: